client/app/dedicated/server/interfaces/interfaces.routing.js

```diff
@@ -18,6 +18,13 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => (translateId, error) => $timeout(() => {
         Alerter.set('alert-danger', $translate.instant(translateId, { error: error.message }));
       }),
+      alertSuccess: /* @ngInject */ (
+        $timeout,
+        $translate,
+        Alerter,
+      ) => translateId => $timeout(() => {
```

Comment:
```suggestion
      ) => message=> $timeout(() => {
```
would be better imo and consistent with the rest (or at least the bits that are)

Reply:
The thing is, `message` is also used for the `error.message` attribute, which brings confusion. That's why I opted for the explicit name `translationId`.